### DISCUSSION TITLE
How to set the width of the wireframe lines

### DISCUSSION THREAD

**Discussion by wddl** [google/filament#7354] [Q&A]
in filament::gltfio::Wireframe class LINES are created , as follows:
`RenderableManager::Builder(1)
        .culling(false)
        .castShadows(false)
        .receiveShadows(false)
        .geometry(0, RenderableManager::PrimitiveType::LINES, mVertexBuffer, mIndexBuffer)
        .build(*engine, mEntity);`
but in android lines width is too  thin.  how to Increase line width? thanks!

**Comment by carloshwa** [comment]
@wddl Were you able to get this to work on Android?
